lib/ticket-types.js

```diff
@@ -2,7 +2,7 @@
 
 function ticketTypes (args, callback) {
   var ticketTypes = [
-    {name: 'ninja', title: 'Ninja', tooltip: 'Select Ninja to create tickets for attendees under 13 or over 13.'},
+    {name: 'ninja', title: 'Youth', tooltip: 'Select Youth to create tickets for attendees under 18.'},
```

Comment:
It actually does mess up with the interface to change it here and in the other pull request - I think there would be a lot more work involved in changing the name key. We might actually change it back eventually anyway, and there are a bunch of places in the schema that we say "ninja"